packages/react/src/components/Provider/Provider.tsx

```diff
@@ -31,12 +37,18 @@ export interface ProviderProps {
 /**
  * The Provider passes the CSS in JS renderer and theme to your components.
  */
-class Provider extends React.Component<ProviderProps> {
-  staticStylesRendered: boolean = false
+class Provider extends UIComponent<ReactProps<ProviderProps>> {
```

Comment:
Once ElementType is replaced (see render) this no longer needs to be extending UIComponent.

👍 thanks Shift